src/docs/introduction/upgrading-to-stencil-three.md

```diff
@@ -224,6 +224,11 @@ export const config: Config = {
 };
 ```
 
+#### Remove `inlineDynamicImports` Configuration
+The `inlineDynamicImports` configuration option on `dist-custom-elements` has been removed. Previously, this option would throw an error at build
+time during the Rollup bundling process if the build contained multiple "inputs" (components). Based on the use cases for Stencil components libraries
```

Comment:
I don't think we need this last sentence, "Based on the use cases...", I don't think it adds much value to the reader. If they're curious as to why we removed this (beyond it being broken),  we can always point them to the PR where we rm'ed it